core/errors.ml

```diff
@@ -186,6 +187,8 @@ let format_exception =
      pos_prefix (Printf.sprintf "Error: Cannot load plugin '%s' (link error: %s)\n" file (Dynlink.error_message err))
   | ClientCallOutsideWebMode fn ->
      pos_prefix (Printf.sprintf "Error: Cannot call client side function '%s' outside of web mode\n" fn)
+  | ClientCallBeforeServerStart fn ->
+     pos_prefix (Printf.sprintf "Error: Cannot call client side function '%s' before server page is ready\n" fn)
```

Comment:
Not to nitpick, but perhaps this and the previous error could be merged into one `ForbiddenClientCall` that takes two parameters, `fn` the function name and `reason` the reason the client call is not possible.

Reply:
Sure, it's better for possible future code reuse.